spec/requests/cache_spec.rb

```diff
@@ -166,19 +166,38 @@ module DisabledAnonymousAPI
     ActionController::Base.allow_forgery_protection = old
   end
 
-  let(:alice) { Fabricate(:account, username: 'alice') }
-  let(:user)  { Fabricate(:user, role: UserRole.find_by(name: 'Moderator')) }
+  # rubocop:disable RSpec/BeforeAfterAll
+  before(:context) do
+    DatabaseCleaner.start
+    create_database_records
+  end
+
+  after(:context) do
+    DatabaseCleaner.clean
+  end
+  # rubocop:enable RSpec/BeforeAfterAll
 
-  before do
-    # rubocop:disable Style/NumericLiterals
-    status = Fabricate(:status, account: alice, id: 110224538612341312)
-    Fabricate(:status, account: alice, id: 110224538643211312, visibility: :private)
+  # rubocop:disable RSpec/InstanceVariable
+  let(:alice) { @alice }
```

Comment:
This looks odd, and is done this way purely to avoid using i-vars in the examples on the page (rubocop complains).